mne/io/nicolet/nicolet.py

```diff
@@ -107,6 +107,15 @@ def _get_nicolet_info(fname, eog, ecg, emg, misc):
                  'ch_names': ch_names, 'description': None,
                  'buffer_size_sec': 10.})
 
+    if ch_type == 'eeg':
+        ch_coil = FIFF.FIFFV_COIL_EEG
+        ch_kind = FIFF.FIFFV_EEG_CH
+    elif ch_type == 'seeg':
+        ch_coil = FIFF.FIFFV_COIL_EEG
+        ch_kind = FIFF.FIFFV_SEEG_CH
+    else:
+        raise TypeError("Channel type not recognized. Available types are "
+                        "'eeg' and 'seeg'.")
     cal = header_info['conversion_factor'] * 1e-6
     for idx, ch_name in enumerate(ch_names):
         if ch_name in eog or idx in eog:
```

Comment:
>
> hmm ... so there are 4 ways to do the same thing? list of indices, list of
> str, tuple of indices, tuple of str?
>

it's more "array-like of int or str"

> and why is list of indices supported? I would think you could get
> something from picks and supply it, but picks is array-like ...
>
sometimes you know that the stim or EOG channel is the last one so it's
easy to set it to -1. Convincing?